README.md

```diff
@@ -87,132 +112,17 @@ likely to find a need for it. If you're creating a number of objects that have s
-`#initialize` which are needed to create instances of itself. In Javascript, there are many ways
-to create objects.
-
-In _classical_ inheritance, it creates a _copy_ of the behavior from parent. Copies of `class`es
-become separate entities, no longer directly associated with the parent. In JavaScript when we
-create the object it does not copy the properties or behavior, it creates a _link_. This is called
-[_prototypal_ inheritance](https://blog.bitsrc.io/understanding-javascripts-prototypal-inheritance-354292253bcb).
-
-> Every object in JavaScript has an internal property called `prototype`. `prototype`s are what
-allows JavaScript objects to inherit features from one another. This is often referred to as a
-_prototype chain_. Methods and properties are not copied from one object to another in the
-prototype chain--they are accessed by _walking up_ the chain.
-
-Let's use `Vehicle` as a parent `class` with `Car` as a child `class` of `Vehicle` as an example:
-
-```js
-class Vehicle { // Parent class
-  constructor(name) {
-    this.name = name;
-  }
-
-  start() { // Parent class method
-    return "Engine of "+this.name + " starting...";
-  };
-}
-
-class Car extends Vehicle { // Child class
- constructor(name) {
-    super(name);
-  }
-
-  run() { // Child class method
-    return `Hello! ${super.start()} It has 4 Cylinders.`; // Calling on a method inherited from parent class
-  }
-}
-
-let car1 = new Car("Camry");
-let car2 = new Car("Civic");
-
-// Accessing the child class method which accesses the parent's method
-car1.run();   // "Hello! Engine of Camry starting... It has 4 Cylinders."
-car2.run();   // "Hello! Engine of Civic starting... It has 4 Cylinders."
-```
-
-While on the surface it may look like `Vehicle` and `Car` do the same thing, `class`es that are
-_linked_ and *not* _copied_ are fundamentally a code reuse mechanism--a way for objects to share
-code. `Car` looks to the parent as a reference, and does not have all `Vehicle`'s properties and
-behaviors without it. The way that the code is shared matters. When you attempt to access a
-property or method of an object, JavaScript will searches through the object/`class` and its
-`prototype` until the end of the `prototype` chain is reached. It can create performance and
-code maintenance issues when misapplied.
-
-### Strong Behavioral Subtyping
-
-When creating child `class`es, we defer to [SOLID](https://en.wikipedia.org/wiki/SOLID). SOLID is
-a set of design principles for OO programming that is intended to make software designs more
-understandable, flexible and maintainable. The 'L' in SOLID stands for is Liskov's substitution
-principle:
-
-> objects in a program should be replaceable with instances of their subtypes without altering
-the correctness of that program.
+People new to Object oriented languages tend to go off the deep end when they
+first see the glory of inheritance: Animals extend MulticellularLife which
```

Comment:
this is some real guru stuff right here

and yes, @drakeltheryuujin i shoulda used `ticks`. shame on me.